calicoctl/commands/resources.go

```diff
@@ -188,6 +189,21 @@ func executeConfigCommand(args map[string]interface{}, action action) commandRes
 			results.err = err
 			break
 		}
+
+		for i, _ := range res {
+			// Remove the cluster specific metadata if the "--export" flag is specified
+			if argutils.ArgBoolOrFalse(args, "--export") {
```

Comment:
I think you can store this in a variable outside the loop so you don't have to keep parsing it for every loop iteration